README.md

```diff
@@ -104,6 +104,12 @@ ${DOCKER_IMAGE}
 
 ```
 
+## Versioning
+
+Semantic versioning is used and containers are tagged based on GitHub tags: If a tag,
+v1.2.3 is pushed to GitHub then a container image is built with tags `1.2.3` as well as `1.2`.
```

Comment:
i assume you mean that both tags are applied to the same build not that the container gets built multiple times with different tags?

Reply:
In `docker_build_push.yml` the `build` job builds the container image once, then the `trivy-scan` job scans the container image once and, finally, the `push-image` job pushes the container image to the registry multiple times, once per tag.

Reply:
why not build with both tags (1.2.3 & 1.2) and then just push once?